rotkehlchen/chain/arbitrum_one/modules/gmx/balances.py

```diff
@@ -159,4 +160,42 @@ def query_balances(self) -> 'BalancesType':
                     usd_value=position_collateral_value,  # it is already given in USD
                 )
 
+    def query_staking_balances(self, balances: 'BalancesType') -> None:
+        """
+        Query staked balances for GMX.
```

Comment:
also in this docstring specify that the staking balances are not returned but modified in-place into the passed balances argument.